CMakeLists.txt

```diff
@@ -5056,6 +5056,8 @@ set(
   "${CMAKE_CURRENT_SOURCE_DIR}/packaging/CPackDebInstall.cmake"
 )
 
+# Use WiX toolset version >= 4.0 for building Windows installers.
+set(CPACK_WIX_VERSION 4)
```

Comment:
We may also asume conributors have newer CMake versions. How is the situation regarding https://cmake.org/cmake/help/latest/policy/CMP0172.html#policy:CMP0172?